--- configure.ac	2024-11-14 17:27:28.996159727 +0100
+++ configure.ac.new	2024-11-14 17:27:24.031143793 +0100
@@ -500,6 +500,7 @@
 		 src/plugins/sched/builtin/Makefile
 		 src/plugins/select/Makefile
 		 src/plugins/select/cons_tres/Makefile
+		 src/plugins/select/cons_tres_org/Makefile
 		 src/plugins/select/linear/Makefile
 		 src/plugins/serializer/Makefile
 		 src/plugins/serializer/json/Makefile

