--- a/doc/man/man1/scontrol.1	2022-05-16 15:55:46.554273450 +0200
+++ b/doc/man/man1/scontrol.1	2022-05-16 15:55:54.462429134 +0200
@@ -494,7 +494,8 @@
 .TP
 \fBjob\fR
 Displays statistics about all jobs by default. If an optional jobid is
-specified, details for just that job will be displayed.
+specified, details for just that job will be displayed. Multiple jobid
+values separated by comma is also possible.
 If the job does not specify socket\-per\-node, cores\-per\-socket or
 threads\-per\-core then it will display '*' in the ReqS:C:T=*:*:* field.
 .IP
diff --git a/src/scontrol/scontrol.c b/src/scontrol/scontrol.c
index cc0c38f25c..7d78d8f95c 100644
--- a/src/scontrol/scontrol.c
+++ b/src/scontrol/scontrol.c
@@ -1791,7 +1791,16 @@
 			exit_code = 1;
 	} else if (xstrncasecmp(tag, "jobs", MAX(tag_len, 1)) == 0 ||
 		   xstrncasecmp(tag, "jobid", MAX(tag_len, 1)) == 0 ) {
-		scontrol_print_job(val, argc, argv);
+		if (val) {
+			char *saved_ptr;
+			char *job_id = strtok_r(val, ",", &saved_ptr);
+			while (job_id != NULL) {
+				scontrol_print_job(job_id, argc, argv);
+				job_id = strtok_r(NULL, ",", &saved_ptr);
+			}
+		} else {
+			scontrol_print_job(val, argc, argv);
+		}
 	} else if (xstrncasecmp(tag, "licenses", MAX(tag_len, 2)) == 0) {
 		scontrol_print_licenses(val, argc, argv);
 	} else if (xstrncasecmp(tag, "nodes", MAX(tag_len, 1)) == 0) {
